ext/tunemygc/tunemygc_ext.c

```diff
@@ -20,6 +21,26 @@ static VALUE sym_gc_cycle_exited;
 #endif
 
 /* From @tmm1/gctools */
+static double _tunemygc_realtime()
+{
+  struct timespec ts;
+#ifdef HAVE_CLOCK_GETTIME
+  if (clock_gettime(CLOCK_REALTIME, &ts) == -1) {
```

Comment:
I thought you need to do something like each machines measure spans of time locally with its monotonic clock, and you piece them back with real time clocks + causality? Or is that naive and real time clocks are the best we can do?